### PR TITLE
wait for Tekton CRDs to be ready

### PR DESCRIPTION
**Pull request by xinredhat**
This PR is to fix bug [RHTAPBUGS-903](https://issues.redhat.com/browse/RHTAPBUGS-903). Sometimes Tekton CRDs need a few mins to be ready, this PR is to make sure all of Tekton CRDs to be ready before running tests.

### CODE REVIEW

**Review by Roming22 — APPROVED** [review]
/lgtm

File: hack/preview.sh [hunk 1]
```diff
@@ -337,3 +337,20 @@ if $KEYCLOAK && $TOOLCHAIN ; then
 	  exit 1
   fi
 fi
+
+# Sometimes Tekton CRDs need a few mins to be ready
+retry=0
+while true; do
+  if [ "$retry" -eq 5 ]; then
+    printf "Error: Tekton CRDs cannot be ready on the cluster.\n" >&2
```
Comment (scoheb) [inline hack/preview.sh:345]:
```suggestion
    printf "Error: Tekton CRDs are not yet available on the cluster.\n" >&2
```

**Pull request merged**